lib/droplet_kit/resources/kubernetes_clusters_resource.rb

```diff
@@ -0,0 +1,64 @@
+module DropletKit
+  class KubernetesClustersResource < ResourceKit::Resource
+    include ErrorHandlingResourcable
+
+    resources do
+      action :all, 'GET /v2/kubernetes/clusters' do
+        query_keys :per_page, :page, :tag_name
+        handler(200) { |response| KubernetesClusterMapping.extract_collection(response.body, :read) }
+      end
+
+      action :find, 'GET /v2/kubernetes/clusters/:id' do
+        handler(200) { |response| KubernetesClusterMapping.extract_single(response.body, :read) }
+      end
+
+      action :create, 'POST /v2/kubernetes/clusters' do
+        body { |object| KubernetesClusterMapping.representation_for(:create, object) }
+        handler(201) { |response, cluster| KubernetesClusterMapping.extract_into_object(cluster, response.body, :read) }
+        handler(422) { |response| ErrorMapping.fail_with(FailedCreate, response.body) }
+      end
+
+      action :config, 'GET /v2/kubernetes/clusters/:id/kubeconfig' do
```

Comment:
Would you mind calling this `:kubeconfig`, mostly for the sake of future-proofing in case we end up having an endpoint to show the cluster configuration in some way.

Reply:
sure